src/node.cc

```diff
@@ -4792,11 +4792,8 @@ inline int Start_TTDReplay(Isolate* isolate, void* isolate_context,
   Environment env(isolate_data, context);
   env.Start(argc, argv, exec_argc, exec_argv, v8_is_profiling);
 
-  bool debug_enabled =
-    debug_options.debugger_enabled() || debug_options.inspector_enabled();
-
   // Start debug agent when argv has --debug
-  if (debug_enabled)
+  if (debug_options.debugger_enabled() || debug_options.inspector_enabled())
```

Comment:
Did we take that change yet or is the hope that I just eliminate it now in preparation for that change?